### PR TITLE
[zh] Sync 1.29 docs and polish informational wording

### PR DESCRIPTION
**Pull request by panpan0000**

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi, this file some more in-synced content from 1.29. Would you like to include them into this PR?
The zh localization doesn't accept partial changes due that may compromise the effect of `./scripts/lsync.sh`.
You can find the difference by running the following command on your `main` branch:

```
./scripts/lsync.sh content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md
```

Or you can directly check the [Official 1.29 Released Docs](https://github.com/kubernetes/website/pull/43082/files) by searching that filename `scheduling-framework.md`.

**Comment by panpan0000** [comment]
@windsonsea  Thanks for your guidance. now it's fixed. thank you for review. now content is in-sync.

File: content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 1]
```diff
@@ -156,15 +156,22 @@ QueueingHint 作为一个回调函数，用于决定是否将 Pod 重新排队
...
+QueueingHint evaluation during scheduling is a beta-level feature.
+The v1.28 release series initially enabled the associated feature gate; however, after the
+discovery of an excessive memory footprint, the Kubernetes project set that feature gate
+to be disabled by default. In Kubernetes {{< skew currentVersion >}}, this feature gate is
+disabled and you need to enable it manually.
+You can enable it via the
 `SchedulerQueueingHints` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
 -->
-在调度过程中对 QueueingHint 求值是一个 Beta 级别的特性，在 1.28 中默认被启用。
+在调度过程中对 QueueingHint 求值是一个 Beta 级别的特性.
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md:167]:
```suggestion
在调度过程中对 QueueingHint 求值是一个 Beta 级别的特性。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 2]
```diff
@@ -156,15 +156,22 @@ QueueingHint 作为一个回调函数，用于决定是否将 Pod 重新排队
...
+v1.28 版本最初开启了这个特性的 feature gate，但是发现了内存占用过多的问题，于是目前该 feature gate 被默认关闭。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md:168]:
```suggestion
v1.28 的系列小版本最初都开启了这个特性的门控；但是发现了内存占用过多的问题，
于是 Kubernetes 项目将该特性门控设置为默认禁用。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 3]
```diff
@@ -156,15 +156,22 @@ QueueingHint 作为一个回调函数，用于决定是否将 Pod 重新排队
...
+在Kubernetes 的 {{< skew currentVersion >}} 版本中，这个 feature gate 是关闭的，你需要手动开启它。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md:169]:
```suggestion
在 Kubernetes 的 {{< skew currentVersion >}} 版本中，这个特性门控被禁用，你需要手动开启它。
```

**Review by windsonsea — COMMENTED** [review]
Thanks. Here are some text suggestions for reference.

**Comment by panpan0000** [comment]
your comments are merged. don't forget to do squash merging :-)

**Comment by windsonsea** [comment]
/lgtm

Let's kick this in.

**Pull request merged**